src/ghost-content/transparent.js

```diff
@@ -32,6 +33,21 @@ module.exports = ({ logger, projectLocation }) => {
       const filePath = path.join(folderPath, file)
       return fs.readFileAsync(filePath, 'utf-8')
     },
+
+    deleteFile: (folder, file) => {
+      const { folderPath } = normalizeFolder(folder)
+      const filePath = path.join(folderPath, file)
+      return Promise.fromCallback(cb => fs.unlink(filePath, cb))
+    },
+
+    directoryListing: (rootFolder, fileEndingPattern = '') => {
+      const flowsDir = normalizeFolder(rootFolder)
+      if (!fs.existsSync(flowsDir)) {
+        return Promise.resolve([])
+      }
+
+      return Promise.fromCallback(cb => glob(`**/*.${fileEndingPattern}`, { cwd: flowsDir }, cb))
```

Comment:
I think you may have an extra dot here, the dot is already part of your normal patterns

Reply:
Do you mean I'd rather remove that dot here and keep it within patterns, or remove it from patterns? )

Reply:
I'd remove it here and keep in patterns for better readability and flexibility (you could make something like `-ui.json` potentially)

Reply:
Done.